content/pricing/oss-project.md

```diff
@@ -0,0 +1,47 @@
+---
+title: OSS Project Pricing
+meta_desc: Pulumi's infrastructure as code SDK and SaaS products together provide plans for all organizations, including OSI licensed  open source projects.
+type: page
+layout: pricing-oss
+
+hero:
+    title: Pulumi Open Source Program
+    description: |
+        At the core of Pulumi is an open source project and an engaged community. Pulumi exists thanks
+        to the work of hundreds of open source contributors around the world. As a company, we are
+        deeply committed to being a good steward of the Pulumi open source project. As a way to
+        give back to the community, the Pulumi Open Source Program was created to help open source
+        projects and teams innovate and scale faster through cloud engineering.
+    cta_text: Request Access
+
+features:
+    title: Program Features
+    description: |
+        Projects can utilize the Pulumi open source version today for free if they prefer to manage
+        their own Pulumi [state store backend](https://pulumi.com/docs/intro/concepts/state/). The
+        Pulumi Open Source Program provides the Pro Tier of the Pulumi Service at no cost to qualifying
+        projects for projects that want a Pulumi managed service that provides features such as dashboards,
+        CICD integrations, role based access controls, and APIs and webhooks. Projects get 25 team
+        members and unlimited project stacks.
+
+requirements:
+    title: Program Requirements
+    description: |
+        **Not seeking profit** --- Your project must be part of an open source foundation that does not seek to make a profit
+        through services. The organization must be a valid 501c3 organization with a Employer
+        Identification Number (EIN).
+
+        **OSI-approved open source license** --- Your project must be published under an OSI-approved open source license.
+
+contact:
+    title: Application Process
+    description: |
+        1. Submit the application form on the right side of this page to help us verify that your project meets all program requirements.
+        2. You can expect an initial response from our team within 5-10 business days. You may be asked to provide additional information
+
+        If your application is accepted, we will work with you to enroll your account into the Program. Your application must be renewed annually to make sure that your project still meets the Program requirements, which may change from year to year. If you no longer qualify for renewal, we will work with you to make any needed transition as smooth as possible.
+
+    form:
+        headline: Request Access
```

Comment:
```suggestion
        headline: Open Source Program Application
```

request access seems really strange as a heading here, this is the application right?

Reply:
Yeah, I think that was copied over from another web form template. Apply is the correct verb.

Reply:
👍